app/components/Skills.js

```diff
@@ -0,0 +1,112 @@
+import Image from "next/image";
+
+import javascriptImg from "../../javascript-logo.png";
+import htmlImg from "../../html-logo.png";
+import cssImg from "../../css-logo.png";
+import reactImg from "../../react-logo-skills.png";
+import nextImg from "../../next-logo.png";
+import tailwindImg from "../../tailwind-logo.png";
+import skillsImg from "../../skills.png";
+import downloadImg from "../../download.png";
+import postgresqlImg from "../../postgresql-logo.png";
+
+export default function Skills() {
+  const skills = [
```

Comment:
I would place this outside of the function. Inside the function, each time this function is called, it will recreate the array object. Placing it outside, will cause it to only create the array once on page load. It's also common, with constant values (or arrays of values), that variable names are all upper case `SKILLS`. You don't have to do that, it's just a common idiom, and not necessarily a hard rule